src/main/java/ch/njol/skript/expressions/ExprEntityOwner.java

```diff
@@ -0,0 +1,107 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.UUIDUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.entity.Tameable;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Entity Owner")
+@Description({
+	"The owner of a tameable entity (i.e. horse or wolf) or a dropped item.",
+	"Getting the owner of a dropped item will only return a loaded entity or a player that has played before. "
+	    + "If the entity was killed, or the player has never played before, will return null.",
```

Comment:
```suggestion
		+ "If the entity was killed, or the player has never played before, will return null.",
```
Replace spaces with tabs